tests/test_flow/test_persistence_compatibility.py

```diff
@@ -25,12 +26,18 @@ def older_serialized_cache_harness(make_counter, tmp_path):
 # To renegerate cache, run the following command from bionic/ dir
 #   `python -m tests.test_flow.generate_test_compatibility_cache`
 def test_caching_compatibility(older_serialized_cache_harness):
-    flow = older_serialized_cache_harness.flow
-    assert flow.get("xy") == 6
-    assert flow.get("yz") == 12
-    assert flow.get("xy_plus_yz") == 18
+    # Auto versioned flow requires Python 3.7 or higher.
```

Comment:
Can we explain why 3.7+ is required? This makes it seem like auto versioning doesn't work in Python 3.6.